docs/StardustDocs/topics/apiLevels.md

```diff
@@ -147,10 +147,10 @@ Currently, we are working on compiler a plugin that generates these properties o
 
 The [Column Accessors API](columnAccessorsApi.md) is a kind of trade-off between safety and needs to be written ahead of
 the execution type declaration. It was designed to better be able to write code in an IDE without a notebook experience. 
-It provides type-safe access to columns but doesn't ensure that the columns really exist in a particular dataframe.
+It provides type-safe access to columns but doesn't ensure that the columns really exist in a particular [`DataFrame`](DataFrame.md).
```

Comment:
not sure if this should be the type or the concept "data frame"

Reply:
I suggest do not differ the concept data frame and the main data type "DataFrame", it leads to misunderstanding in the documentation.